base_location_geonames_import/models/res_country.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Franco Tampieri, Freelancer http://franco.tampieri.info
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCountryState(models.Model):
+
+    _inherit = 'res.country'
+
+    geonames_state_name = fields.Integer(
```

Comment:
I think the fields can be named "geonames_state_name_column" and "geonames_state_code_column". This avoids misunderstanding.

Reply:
Well, I suggested the contrary because the long names, but now I don't have a strong opinion on this.

Reply:
I have no problem to restore the old field names

Reply:
OK, what you prefer